src/lib/manual.json

```diff
@@ -78,6 +78,18 @@
       "username": "brenoprata10",
       "repo": "nvim-highlight-colors",
       "tags": ["color"]
+    },
+    {
+      "type": "github",
+      "username": "andymass",
+      "repo": "vim-matchup",
+      "tags": [
+        "keybinding",
+        "programming-languages-support",
+        "motion",
+        "treesitter-based",
+        "text-objects"
```

Comment:
let's not create a new tag

Reply:
I like plugins which provide new text objects and I think there are quite a few and the name of a plugin does not gives this away. Are you strongly against the tag `text-objects`?

For vim there is following website which lists many possible ones: https://github.com/kana/vim-textobj-user/wiki

Reply:
I'm not against it but most of these tags are derived from https://github.com/rockerBOO/awesome-neovim so modifying them might cause conflicts with other names.